packages/common/src/utils/trackUtils.ts

```diff
@@ -0,0 +1,45 @@
+import { DogEarType } from 'models/DogEar'
+import { PremiumConditions } from 'models/Track'
+
+import { Nullable } from './typeUtils'
+
+type GetDogEarTypeArgs = {
+  doesUserHaveAccess?: boolean
+  isArtistPick?: boolean
+  isOwner?: boolean
+  isUnlisted?: boolean
+  premiumConditions?: Nullable<PremiumConditions>
+}
+
+export const getDogEarType = ({
```

Comment:
Since we also have dog ear for collections, can we specify this to either handle both, or just say this is track related?

Reply:
I think this already handles both - correct me if I'm wrong but I grepped `DogEar` in `web` and I believe I've changed every instance. Only saw one collection-related instance on `CollectionPage` that was only used for `HIDDEN` so that case is covered.
Maybe I should just change the file name to `dogEarUtils.ts`.